docs/README.md

```diff
@@ -17,35 +17,370 @@ with a simple and easy to use interface for working with files stored in the Alf
+2. [Search](#search)
+3. [Current User](#current-user)
+
+![](images/header.png)
+
+#### Logo and Color
+
+Logo & app primary color - logo and color are configurable by updating the
+[app.config.json](https://github.com/Alfresco/alfresco-content-app/blob/master/src/app.config.json) file in the root folder of the project.
+Please refer to the [Application Configuration](https://github.com/Alfresco/alfresco-content-app/blob/master/docs/configuration.md#application-logo) documentation for more information on how to change the logo and color.
+
+#### Search
+
+The application [Search](https://github.com/Alfresco/alfresco-content-app/tree/master/src/app/components/search) -
+uses the [ADF Search Component](https://github.com/Alfresco/alfresco-ng2-components/tree/master/lib/content-services/search)
+the app provides a 'live' search feature, where users can open files and folders directly from the Search API results.
+
+![](images/search.png)
+
+#### Current User
+
+[Current User](https://github.com/Alfresco/alfresco-content-app/tree/development/src/app/components/current-user) -
+displays the user's name, and a menu where users can logout.
+Optionally through updating the [app.config.json](https://github.com/Alfresco/alfresco-content-app/blob/master/src/app.config.json)
+a language switching menu can be displayed.
+
+![](images/current-user.png)
+
+### Side Navigation
+
+The application [side navigation](https://github.com/Alfresco/alfresco-content-app/tree/master/src/app/components/sidenav) has two features:
+a button menu and navigation links.
+
+![](images/side-nav.png)
+
+#### New button
+
+The New button displays a menu which provides three actions:
+
+- Create a new folder - provides a dialog which allows the creation of a new folder, the folder name is mandatory and the description is optional.
+- Upload a file - invokes the operating system file browser and allows a user to select file(s) to upload into their current location in the content repository.
+- Upload a folder - invokes the operating system folder browser and allows a user to select a folder to upload to their current location in the content repository.
+
+When an upload starts the [upload component](https://github.com/Alfresco/alfresco-ng2-components/tree/master/lib/content-services/upload)
+is displayed which shows the user the progress of the uploads they have started.
+The upload dialog persists on the screen and can be minimized; users are able to continue using the application whilst uploads are in progress
+and uploads can be canceled which will stop uploads in progress or permanently delete already completed uploads.
+
+![](images/uploader.png)
+
+#### Navigation
+
+The navigation links are configurable via the [app.config.json](https://github.com/Alfresco/alfresco-content-app/blob/master/src/app.config.json).
+Default configuration creates two sections.
+See [Navigation](#navigation) for more information about configuring the side navigation.
```

Comment:
Denys, this link seems to be broken. Please check.

Reply:
Navigation is a relative link within the page when docs are rendered. Should work fine when deployed but I will re-check.

Reply:
ok, thanks